userguide/snippets/alertevents.rst

```diff
@@ -110,6 +110,8 @@ Some of the conditions that trigger an alert include:
 * the interface which is set as critical for failover is not found
   or is not configured
 
+* NVDIMM problems are detected
```

Comment:
The introductory sentence before this list is "Some of the conditions that trigger an alert include:" Thus, 
```suggestion
* NVDIMM problems
```
would make more sense.